tests/unit/test4_diag_print/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(test4_diag_print)
+include(../../cmake/test_template.cmake)
+
+FILE(GLOB zcbor_sources ../../../src/*.c)
+target_sources(app PRIVATE
+  ${zcbor_sources})
+
+target_include_directories(app PRIVATE ../../../include)
```

Comment:
What about assigning `../../../` to a variable?

Reply:
Done